ui/app/templates/components/tool-unwrap.hbs

```diff
@@ -11,10 +11,7 @@
     <nav class="tabs has-bottom-margin-l">
       <ul>
         {{! template-lint-configure no-nested-interactive "warn" }}
-        <li
-          aria-selected={{if (eq @unwrapActiveTab "data") "true" "false"}}
```

Comment:
This was causing a11y lint error. We should replace this with HDS tabs or the like soon